images/kernel/README.md

```diff
@@ -0,0 +1,41 @@
+# Kernel Images
+
+These kernel OCI images contain the kernel binary (at `/boot/vmlinux`) and supporting modules (in `/lib/modules`)
+for guest VMs ran by Ignite.
+
+## Building the Kernel Images
+
+```console
+$ make
+```
+
+## Versions
+
+All LTS versions starting from 4.14 and above are supported by the Ignite team.
+This means in practice:
+
+- 4.14.x
+- 4.19.x
+- 5.4.x
+
+The exact patch versions may be found in the [Makefile](Makefile).
+The available versions exist in the [stable kernel git tree](https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/refs/).
+
+## Upgrading to a new kernel version
+
+It should be as easy as:
+
+```console
+$ make upgrade
+```
+
+after you've upgraded the values in the Makefile.
```

Comment:
```suggestion
after you've set new `KERNEL_VERSIONS` in the Makefile or changed `./config-patches`.
This will regenerate and produce new patched, versioned configs.

Older kernel versions should be deleted from `generated/` and `versioned/` when checking in new ones.
```